go.mod

```diff
@@ -15,7 +15,6 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/mdlayher/vsock v1.1.1
-	github.com/pkg/errors v0.9.1
```

Comment:
Did this not affect `go.sum`?

Reply:
Since our transitive dependencies are still using pkg/errors, yes. Go still need to record the information about the package in go.sum.